src/main/java/seedu/address/logic/commands/ExitCommand.java

```diff
@@ -9,6 +9,7 @@
 public class ExitCommand extends Command {
 
     public static final String COMMAND_WORD = "exit";
+    public static final String COMMAND_ALIAS = "ext";
```

Comment:
"esc" might be another possible alias?

Reply:
Good idea! But it may confuse the user since "esc" is short for escape

Reply:
Make sense. I think we should stick to "ext".